api/v1/lib/agent/calls/calls.go

```diff
@@ -3,7 +3,7 @@ package calls
 import (
 	"time"
 
-	"github.com/mesos/mesos-go/api/v1/lib"
+	mesos "github.com/mesos/mesos-go/api/v1/lib"
```

Comment:
nit: we don't do this anywhere else, why here?

Reply:
No real reason, I use vscode and it keeps on updating it whenever I save the file, I'll revert.